test/main/java/com/liveramp/hyperminhash/CommonTests.java

```diff
@@ -206,13 +201,32 @@ private static double getError(long result, long expected) {
     }
   }
 
-  private static String randomString(final Random random) {
+  private static byte[] randomByteArray(final Random random) {
     final int n = Math.abs(random.nextInt()) % 32;
     byte[] b = new byte[n];
-    for (int i = 0; i < n; i++) {
-      b[i] = (byte) LETTER_BYTES.charAt(randPositiveInt(random) % LETTER_BYTES.length());
-    }
-    return Hex.encodeHexString(b);
+    random.nextBytes(b);
+    return b;
+  }
+
+  public static final byte[] intToByteArray(int value) {
```

Comment:
removed all usages of strings in the test class since strings are super slow and we were generating a ton of them. Replaced with byte arrays or longs/ints for a ~6x speedup in test performance